Reyes_JohnRuiz/script.js

```diff
@@ -0,0 +1,74 @@
+document.addEventListener("DOMContentLoaded", function () {
+  const commentNameInput = document.getElementById("comment-name");
+  const commentTextInput = document.getElementById("comment-text");
+  const submitCommentButton = document.getElementById("submit-comment");
+  const commentsContainer = document.getElementById("comments-container");
+  const sortAscButton = document.getElementById("sort-asc");
+  const sortDescButton = document.getElementById("sort-desc");
+
+  let comments = [];
+
+  function checkCommentValidity() {
+    const nameValue = commentNameInput.value.trim();
+    const textValue = commentTextInput.value.trim();
+    submitCommentButton.disabled = !(nameValue && textValue);
+  }
+
+  checkCommentValidity();
+
+  commentNameInput.addEventListener("input", checkCommentValidity);
+  commentTextInput.addEventListener("input", checkCommentValidity);
+
+  submitCommentButton.addEventListener("click", (e) => {
```

Comment:
Do not use arrow function as a method of an object. You can use the sample code and adjust it to your intended use
```suggestion
  submitCommentButton.addEventListener("click", function() => {
```